src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -201,7 +201,17 @@ class Storage extends React.Component {
               id={`${idPrefix}-${value}-size-edit`}
               type='number'
               defaultValue={sizeGiB}
-              onBlur={e => this.handleCellChange(rowData, 'size', e.target.value)}
+              onFocus={() => { this.isDiskSizeFocused = true }}
+              onBlur={e => {
+                this.isDiskSizeFocused = false
+                this.handleCellChange(rowData, 'size', e.target.value)
+              }}
+              onChange={e => {
+                if (!this.isDiskSizeFocused) {
+                  this.handleCellChange(rowData, 'size', e.target.value)
```

Comment:
Please note that the down arrow is not disabled when 0 value is set (and after clicking the -1 value is set).
In general, negative values are not allowed to disk size either by clicking arrows or by entering the text.

Reply:
I wasn't focused in the field validation, I assumed the  validation is all right, I'll be more critical next time
Thanks